src/StoreApi/Schemas/CartItemSchema.php

```diff
@@ -450,6 +464,38 @@ protected function get_item_data( $cart_item ) {
 		return array_map( [ $this, 'format_item_data_element' ], $item_data );
 	}
 
+	/**
+	 * Return the minimum quantity that's allowed for this item.
+	 *
+	 * @param array $cart_item Cart item array.
+	 * @return number
+	 */
+	protected function get_item_quantity_min( $cart_item ) {
+		/**
+		 * Filters the quantity minimum for a cart item in Store API.
+		 *
+		 * @param array $cart_item Cart item array.
+		 * @return number
+		 */
+		return apply_filters( 'woocommerce_store_api_item_quantity_minimum', 1, $cart_item );
```

Comment:
These filters use $cart_item, `woocommerce_store_api_product_quantity_limit` uses $product. Which is correct?

Reply:
I see the other is product endpoint only. We should make these filters consistent.